reference_docs/YAML_models/iec_document_attributes.yaml

```diff
@@ -0,0 +1,111 @@
+version: v3.4.5 #Verify versions
+
+inherits_from:
+  # Also may be referred to as “parent flavor”
+  flavor: standoc
+  version: v1.x.x
+
+document_attributes:
+#--- Parts and subparts  ---#
+  partnumber:
+    tags: [""]
+    title: "Part number"
+    description: |
+      "IEC documents can have both parts and subparts, unlike ISO documents which only supports parts.\n
+      Both `{part}` and `{subpart}` only accept numerals.".
+    value_spec: "other: `{part}` or `{part}`-`{subpart}`"
+
+#--- Document relations ---#
+  translated-from:
+    tags: ["Document relations"]
+    title: "Translated from"
+    description: |
+      "Document identifier that the current document is a translation of.
+
+      IEC documents can be translated by IEC-CO and tagged with the `translated-from` attribute. This is used to derive the correct status code.".
+    more_information: "See link:/author/ref/document-attributes[general Metanorma document attributes] for more information."
+
+#--- Document type ---#
+  doctype:
+    tags: ["Document type"]
+    title: "Document type"
+    description: |
+      "Type of document.\n
+      Unlike ISO, IEC does not have `international-workshop-agreement`, and does have `interpretation-sheet`.".
+    value_spec: "enum: international-standard, technical-specification, technical-report, publicly-available-specification, guide, interpretation-sheet"
+  required: true
+
+
+#--- Project stage and substage of document ---#
+  docstage:
+    tags: [""]
+    title: "Document stage"
+    description: "Project stage code of document".
+    more_information: |
+      "The codes in the first entry form are drawn from the
+      https://www.iso.org/stage-codes.html[ISO International Harmonized Stage Codes].
+
+      The mapping to IEC stage codes is taken from
+      https://www.iec.ch/members_experts/refdocs/iec/isoiecdir1-consolidatediecsup%7Bed15.0.RLV%7Den.pdf[ISO/IEC DIR 1:2019 + IEC SUP:2019, Annex SH],
+      updated against the current discussion of stage codes in
+      https://www.iec.ch/standardsdev/resources/processes/stage_codes.htm[IEC Processes & Procedures -- Stage Codes] and
+      https://www.iec.ch/standardsdev/resources/processes/workflows.htm[IEC Processes & Procedures -- Workflow]."
+    required: true
+
+  docsubstage:
+    tags: ["Project stage and substage of document"]
+    title: "Document substage"
+    description: "Project substage code of document".
+
+  stage:
```

Comment:
`:docstage:` uses the numeric codes which ISO has defined as applicable across all ISO/IEC documents, as a universal general encoding.

`:stage:` uses the abbreviations for the stage that are specific to IEC, and which incorporate  document stage and substage. We at metanorma like docstage, because they follow a more generic standard, and in fact, we map stage to docstage anyway. But people going through the IEC document process are going to be exposed to the IEC `stage` abbreviations, since they are native to IEC.